src/main/java/de/retest/recheck/persistence/JUnitTestbasedShortNamingStrategy.java

```diff
@@ -0,0 +1,23 @@
+package de.retest.recheck.persistence;
+
+/**
+ * Extends the standard {@link JUnitTestbasedNamingStrategy}, but instead of the qualified class name (e.g.
+ * `com.mycompany.MyTest`) returns the {@link Class#getSimpleName()} of the class (e.g. `MyTest`).
+ */
+public class JUnitTestbasedShortNamingStrategy extends JUnitTestbasedNamingStrategy {
+
+	/**
+	 * @return The {@link Class#getSimpleName()} of the test class (e.g. `MyTest`).
+	 */
+	@Override
+	public String getSuiteName() {
+		String result = super.getSuiteName();
+		if ( result.contains( "." ) ) {
+			result = result.substring( result.lastIndexOf( "." ) + 1 );
+		}
+		if ( result.contains( "$" ) ) {
```

Comment:
Do we need to handle inner classes? Is this possible ...
Don't you still expect something like: `InnerClass`

```java
public static class Foo {

	public static class Bar {}
}

Foo.Bar f = new Foo.Bar();
System.out.println(f.getClass().getSimpleName());
```

Output: Bar

Reply:
Just for naming the checks, I would ignore inner classes. Should be a corner case anyways...

Reply:
If someone wants to change that, they can create a PR.